test/built-ins/Object/Assign/LengthOfAssign.js

```diff
@@ -0,0 +1,7 @@
+/*---
+description: The length property of the assign method should be 2
+es6id:  19.1.2.1
+author: Wei Guo
+---*/
+
+assert(Object.assign.length === 2 , "The length property of the assign method should be 2");
```

Comment:
Can you also test the property descriptor? You can use the `propertyHelper.js` file--see [the test for `Set.length`](https://github.com/tc39/test262/blob/527644d21199d5e9fcab0b958c3426cbd0d0b9d6/test/built-ins/Set/length.js) for an example